include/api/wasmedge/int128.h

```diff
@@ -15,7 +15,7 @@
 #ifndef WASMEDGE_C_API_INT128_H
 #define WASMEDGE_C_API_INT128_H
 
-#if defined(__x86_64__) || defined(__aarch64__)
+#if defined(__x86_64__) || defined(__aarch64__) || defined(__riscv)
```

Comment:
I am not sure if the `__riscv` means `risc-v 64` only. If not, we should not use this directly.
The 32 bits and 64 bits are totally different cases. And we should separate them into different conditions. Also, you may need to check if `uint128_t` and `int128_t` are supported on these platforms.

Reply:
In fact, there is a definition in `/opt/riscv64/include/c++/11.1.0/riscv64-unknown-linux-gnu/bits/c++config.h` which is used by `riscv-gnu-toolchain`:
```
/* Define if __int128 is supported on this host. */
#define _GLIBCXX_USE_INT128 1
```
And I write the `1.cpp` for test:
```cpp
#include<iostream>
using namespace std;
int main(){
	#if __SIZEOF_INT128__
	cout<<"yes"<<endl;
	#else
	cout<<"no"<<endl;
	#endif
	return 0;
}
```
Then I test it:
```bash
(base) wcy@wcy-Lenovo-XiaoXinAir-14IML-2019:~/test$ riscv64-unknown-linux-gnu-g++ 1.cpp -o 1.o
(base) wcy@wcy-Lenovo-XiaoXinAir-14IML-2019:~/test$ qemu-riscv64 -L /opt/riscv64/sysroot  1.o
yes
```
I'm not sure if it means that the riscv-64 support the `int128_t` or not?

Reply:
Yes. I think so